configure.ac

```diff
@@ -109,7 +108,7 @@ AC_ARG_ENABLE(profiling,      [  --enable-profiling        enable profiling with
 	  
 CFLAGS="${generic_CFLAGS} ${ext_CFLAGS} ${user_CFLAGS} ${debug_CFLAGS} ${OPENMP_CFLAGS}"
 CXXFLAGS="$CFLAGS"
-CXXFLAGS="${CXXFLAGS} ${BOOST_CPPFLAGS} ${BAM_CPPFLAGS} ${EIGEN3_CPPFLAGS}"
```

Comment:
please don't use hacky m4 macros, and instead rely on pkg-config. Pkg-config is faster, scales better, is easier to get to work with cross-compiling, and avoids guessing upstream dependencies. The problem here is that `PKG_CHECK_MODULES` creates `x_CFLAGS` as precious variable, not `x_CPPFLAGS`, hence, this could never have worked in the first place.

https://autotools.io/pkgconfig/pkg_check_modules.html

Reply:
If you weren't just indulging in drive-by ideology, you would know that pkg-config was temporarily used for this (PR #27) but the maintainers found it didn't suffice for their needs (537355dbaa29da55168148b25a224730dd42594e) so reverted it. This reversion was incomplete (#68) hence this pull request to make cufflinks build again.

You would also have noticed that these issues are nearly three years old and nothing has been committed to this repository for 18 months, so the question is mostly moot.